src/app/GlobalRedux/Features/MarketTableSlice/index.ts

```diff
@@ -1,15 +1,14 @@
 `use client`;
 
 import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
-import { Coin } from "../../../../../interfaces";
 
 export const fetchCoins = createAsyncThunk(
   "coins/getCoins",
   async (currency: string, thunkApi) => {
     const COIN_URL = `https://api.coingecko.com/api/v3/coins/markets?vs_currency=${currency}&order=market_cap_desc&per_page=20&page=1&sparkline=true&price_change_percentage=1h%2C24h%2C7d&locale=en`;
     const response = await fetch(COIN_URL);
     const data = await response.json();
```

Comment:
consider mazking your own api function so you don;t repeathe these two lines all the time

Reply:
moving this suggestion to the backlog, I agree that's a better way to handle it.